scripts/setup.py

```diff
@@ -214,6 +214,12 @@ def main(args=None):
     elif os.path.isfile('/usr/bin/chromium-browser'):
         # Unix.
         chrome_bin = '/usr/bin/chromium-browser'
+    elif os.path.isfile('/usr/bin/google-chrome-beta'):
+        # Unix.
+        chrome_bin = '/usr/bin/google-chrome-beta'
+    elif os.path.isfile('/usr/bin/chromium'):
```

Comment:
How is this different from line 214?

Reply:
"chromium-browser" is changed to only "chromium" 
checked on arch linux.

Reply:
Can you comment that?